style.css

```diff
@@ -509,6 +514,520 @@ textarea { resize: none; }
+    flex-wrap: wrap;
+  }
+
+  .about-heading {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .about-description {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #42526e;
+    width: fit-content;
+    height: auto;
+  }
+
+  .about-button {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    padding: 20px;
+    background: #36b37f;
+    border-radius: 6px;
+    border-color: #fff;
+    color: #fff;
+    margin-top: 15px;
+    align-items: center;
+  }
+
+  .about-card-box {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 15px;
+    width: auto;
+    justify-content: center;
+  }
+
+  .boxes {
+    display: grid;
+    justify-items: center;
+    background: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    margin: 5px;
+    min-width: 200px;
+  }
+
+  .icon-logo {
+    display: flex;
+    justify-items: center;
+    align-self: center;
+    margin-top: 30px;
+  }
+
+  .topic {
+    width: fit-content;
+  }
+
+  .topic h3 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 20px;
+    line-height: 32px;
+    color: #344563;
+    height: fit-content;
+    margin-top: 50px;
+    min-width: fit-content;
+  }
+
+  .list-about {
+    display: flex;
+    flex-flow: wrap;
+    align-items: center;
+    justify-content: center;
+    border-radius: 8px;
+    margin-top: 60px;
+  }
+
+  .list-tag {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    background: #fff;
+    color: #36b37e;
+    padding: 10px 12px;
+    border-radius: 8px;
+  }
+
+  .contact-footer {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    height: 500px;
+    min-width: auto;
+    padding: 30px;
+    border-bottom: 2px solid #dfe1e6;
+    background-image:
+      url(./Images/illustration-1-contact-form-desktop.svg),
+      url(./Images/illustration-2-contact-form-desktop.svg),
+      url(./Images/contact-form-illustratoin-3-desktop.svg);
+    background-repeat: no-repeat;
+    background-position: 0 230px, 350px 400px, right;
+  }
+
+  .contact-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  .contact-head {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .contact-class {
+    display: flex;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+    max-width: 500px;
+    height: 300px;
+    align-items: center;
+  }
+
+  form {
```

Comment:
- [ OPTIONAL] I just noticed that you forgot to place a 'line-bar' on the bottom of your mobile version, which was not noted in the previous evaluation. Kindly fix this so that your project matches the Figma design. :upside_down_face:  :smile: 
Figma
![linebar](https://user-images.githubusercontent.com/88894525/194711895-dac1b07d-f9f1-47db-ae2d-3c6cb08cdf88.png)

Your Design
![arrang3](https://user-images.githubusercontent.com/88894525/194712043-86cd7c68-250c-428f-838d-ca7c36ab4e94.png)